salt/orchestrate/edx/templates/ansible_env_config.yml.j2

```diff
@@ -0,0 +1,533 @@
+        collection: 'modulestore'
+    direct:
+      ENGINE: "{{ MONGODB_MODULESTORE_ENGINE }}"
+      OPTIONS:
+        collection: "modulestore"
+        db: "{{ MONGODB_MODULESTORE_DB }}"
+        default_class: "xmodule.raw_module.RawDescriptor"
+        fs_root: "{{ GIT_REPO_DIR }}"
+        host:
+          - "{{ MONGODB_HOST }}"
+        password: "{{ MONGODB_MODULESTORE_PASSWORD }}"
+        port: "{{ MONGODB_PORT }}"
+        render_template: "edxmako.shortcuts.render_to_string"
+        user: "{{ MONGODB_MODULESTORE_USER }}"
+      DOC_STORE_CONFIG:
+        db: "{{ MONGODB_MODULESTORE_DB }}"
+        host:
+          - "{{ MONGODB_HOST }}"
+        password: "{{ MONGODB_MODULESTORE_PASSWORD }}"
+        port: "{{ MONGODB_PORT }}"
+        user: "{{ MONGODB_MODULESTORE_USER }}"
+        collection: 'modulestore'
+  OPEN_ENDED_GRADING_INTERFACE:
+    grading_controller: "grading_controller"
+    password: "{{ XQUEUE_PASSWORD }}"
+    peer_grading: "peer_grading"
+    staff_grading: "staff_grading"
+    url: "http://localhost:8060/"
+    username: "{{ XQUEUE_USER }}"
+  PEARSON_TEST_PASSWORD: ""
+  SECRET_KEY: "{{ CMS_DJANGO_SECRET_KEY }}"
+  XQUEUE_INTERFACE:
+    # basic_auth:
+    #   - "mitx"
+    #   - "{{ XQUEUE_PASSWORD }}"
+    django_auth:
+      password: "{{ XQUEUE_PASSWORD }}"
+      username: "{{ XQUEUE_USER }}"
+    url: "http://localhost:8040"
+    callback_url: "http://localhost:8199"
+  YOUTUBE_API_KEY: "{{ YOUTUBE_API_KEY }}"
+  ZENDESK_API_KEY: ""
+  ZENDESK_USER: ""
+
+cms_env_config: &cms_env
+  'ADMINS' :
+  - ['{{ ADMIN_EMAIL_NAME }}', '{{ ADMIN_EMAIL }}']
+  BUGS_EMAIL: "{{ BUGS_EMAIL }}"
+  CONTACT_EMAIL: "{{ CONTACT_EMAIL }}"
+  TECH_SUPPORT_EMAIL: "{{ TECH_SUPPORT_EMAIL }}"
+  ANALYTICS_SERVER_URL: ""
+  BOOK_URL: ""
+  CACHES:
+    celery:
+      BACKEND: "django.core.cache.backends.memcached.MemcachedCache"
+      KEY_FUNCTION: "util.memcache.safe_key"
+      KEY_PREFIX: "integration_celery"
+      LOCATION:
+        - "localhost:11211"
+    default:
+      BACKEND: "django.core.cache.backends.locmem.LocMemCache"
+      KEY_FUNCTION: "util.memcache.safe_key"
+      KEY_PREFIX: "sandbox_default"
+      LOCATION: "mitx_loc_mem_cache"
+    general:
+      BACKEND: "django.core.cache.backends.dummy.DummyCache"
+      KEY_FUNCTION: "util.memcache.safe_key"
+      KEY_PREFIX: "general"
+      LOCATION:
+        - "localhost:11211"
+      VERSION: 4
+    mongo_metadata_inheritance:
+      BACKEND: "django.core.cache.backends.filebased.FileBasedCache"
+      KEY_FUNCTION: "util.memcache.safe_key"
+      KEY_PREFIX: "integration_mongo_metadata_inheritance"
+      LOCATION: "/var/tmp/mongo_metadata_inheritance"
+      TIMEOUT: 300
+    staticfiles:
+      BACKEND: "django.core.cache.backends.memcached.MemcachedCache"
+      KEY_FUNCTION: "util.memcache.safe_key"
+      KEY_PREFIX: "integration_static_files"
+      LOCATION:
+        - "localhost:11211"
+  CELERY_BROKER_HOSTNAME: "localhost"
```

Comment:
Is this just a default value that gets overridden somewhere else? (I haven't read through the whole file yet...)